tex/main.tex

```diff
@@ -36,6 +75,8 @@ \section{Introduction}\label{sec:introduction}
     \item Short discussion about the Axelrod library.
 \end{itemize}
 
+I'm happy to write this section. We can lift some references from one of my papers on the Moran process.
```

Comment:
Make sense, I kind of threw this (and those two citations) just to get the structure up :)